pipeline/extractor/src/main/scala/org/sunbird/obsrv/extractor/functions/ExtractionFunction.scala

```diff
@@ -37,26 +40,34 @@ class ExtractionFunction(config: ExtractorConfig, @transient var dedupEngine: De
                               context: ProcessFunction[mutable.Map[String, AnyRef], mutable.Map[String, AnyRef]]#Context,
                               metrics: Metrics): Unit = {
     metrics.incCounter(config.defaultDatasetID, config.totalEventCount)
+    if(batchEvent.contains(Constants.INVALID_EVENT)) {
```

Comment:
Handle with the error code from the obsrv_meta

Reply:
updated the code